src/01_bignum.py

```diff
@@ -1,4 +1,6 @@
 # Print out 2 to the 65536 power
 # (try doing the same thing in the JS console and see what it outputs)
 
-# YOUR CODE HERE
\ No newline at end of file
+# YOUR CODE HERE
+
+print(pow(2, 65536))
```

Comment:
Very nice! Nearly everyone else did print(2 ** 65536). I like that you used the method for it though.